pkg/convert/byteConverter.go

```diff
@@ -48,3 +48,33 @@ func BytesToString(entry []byte) string {
 
 	return string(content)
 }
+
+func ByteToStringArray(entry []byte) []string {
+	var result []string
+
+	var stringContent string
```

Comment:
Should be defined inside the for loop to reduce variable scope and this variable name is very poor...